plugins/pulp_rpm/plugins/profilers/yum.py

```diff
@@ -287,8 +287,9 @@ def _get_rpms_from_errata(errata):
         """
         rpms = []
         if "pkglist" not in errata.metadata:
-            _logger.warning(_("metadata for errata <%s> lacks a 'pkglist'") % (
-                            errata.unit_key.get('errata_id')))
+            msg = _("metadata for errata <%(errata_id)s> lacks a 'pkglist'")
+            msg_dict = {'errata_id': errata.unit_key.get('errata_id')}
+            _logger.warning(msg % msg_dict)
```

Comment:
Oh yeah I just tested that and it does. I'm pushing a new commit that includes only this change and then I'm merging. Thanks @jortel